.editorconfig

```diff
@@ -0,0 +1,13 @@
+[*]
```

Comment:
Should .editorconfig be part of pr? maybe it should be in .gitignore?

Reply:
I believe adding this is the purpose of the MR?  We want the `.editorconfig` in the repo to fix the mis-formatting of .md files.